app/services/time_of_need_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'restforce'
+require 'faraday'
+
+class TimeOfNeedService
+
+  @conn = Faraday.new(:url => 'https://va--mbmssit.cs33.my.salesforce.com/services/oauth2/token')
+
+  @request = @conn.post do |req|
+    req.headers['Content-Type'] = 'application/x-www-form-urlencoded'
+    req.body = 'client_secret=6150175763295809569&client_id=3MVG9Zdl7Yn6QDKPHMwHadV4iFddvfNHNDYs18l_CHPw5JkdgIjR0trqqefjrP4jbfu5AHnCbxyEg60Tz3KWW&api_version=42.0&username=mbms_integration@bah.com.mbmssit&password=littlekyle2spCL3xcR7dbcUrrSwcgoUrDQ&grant_type=password'
```

Comment:
Client Secret should NOT be stored here, please follow patterns put in place already